ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -71,14 +71,15 @@ export default function CustomSpendingCap({
       return {
         className: 'custom-spending-cap__lowerValue',
         description: t('inputLogicEqualOrSmallerNumber', [
-          <Typography
+          <Text
             key="custom-spending-cap"
-            variant={TypographyVariant.H6}
+            variant={TextVariant.bodySm}
+            as="h6"
             fontWeight={FONT_WEIGHT.BOLD}
```

Comment:
You can use the `bodySmBold` variant here
```suggestion
            variant={TextVariant.bodySmBold}
            as="h6"
```